pkg/controller/cluster/controller.go

```diff
@@ -17,17 +17,30 @@ limitations under the License.
 package cluster
 
 import (
+	"fmt"
+	"time"
+
 	"github.com/kubernetes-incubator/apiserver-builder/pkg/builders"
+	"github.com/kubernetes-incubator/apiserver-builder/pkg/controller"
 
 	"github.com/golang/glog"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
 	clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 	"sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset"
 	listers "sigs.k8s.io/cluster-api/pkg/client/listers_generated/cluster/v1alpha1"
 	"sigs.k8s.io/cluster-api/pkg/controller/sharedinformers"
 	"sigs.k8s.io/cluster-api/pkg/util"
 )
 
+type RequeueAfterError struct {
```

Comment:
exported things need docs

Reply:
Should we move the machine controller's RequeueAfterError somewhere more common and re-use for both controllers?

Reply:
Added a doc string and moved to a new shared package